util/Cargo.toml

```diff
@@ -44,7 +44,7 @@ path = "src/pem_to_public_dnskey.rs"
 
 [dependencies]
 clap = "^2.23.3"
-data-encoding = "^1.2.0"
+data-encoding = "^2.0.0"
```

Comment:
Can you just let me know what we're getting with this upgrade?

Reply:
The changes between 1.2.0 and 2.0.0 that are relevant for this project are:
- Support for character translation (e.g. for case-insensitivity).
- Support for unpadded encodings (support in 1.2.0 was adhoc).
- Exact same performance for encodings defined in child crates as for those defined in `data-encoding` itself. With 1.2.0, custom encodings used to be slower.

However, the current patch does not use any of these directly. Using `lazy_static` or waiting for `2.1.0` would. I mainly do this change as part of helping my users migrate through the API break from v1 to v2.